SubmitUserActivites.cs

```diff
@@ -0,0 +1,342 @@
+using System.Net;
+using FirebaseAdmin.Auth;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using MySqlConnector;
+using Newtonsoft.Json;
+
+namespace Company.Function
+{
+    public class SubmitUserActivites
+    {
+        public SubmitUserActivites()
+        {
+            FirebaseInitializer.Initialize();
+        }
+
+        [Function("SubmitUserActivites")]
+        public static async Task<IActionResult> SubmitInformationAsync([HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "submit-user-activity")] HttpRequestData req)

```

Comment:
Can you create test for this function? It does need to be passing all because Database is not setup yet, but have test for ArgumentException, FirebaseAuthException.

Also, can you use HttpResponseData instead of IActionResult. You can check how I use it in CreateUser.

Reply:
Changed input property name of the request from e.g. {TransportType: <string>, Miles: <double>} to {Type: <string>, Amount: <double>} for all activities